docs/faq.rst

```diff
@@ -160,6 +160,12 @@ How can I use PyScaffold if my project is nested within a larger repository, e.g
     In future versions of PyScaffold this will be much simpler as ``pyproject.toml`` will completely replace ``setup.py``.
 
 
+What is the license of the generated project scaffold? Is there anything I need to consider?
+    The source code of PyScaffold itself is MIT-licensed with the exception of the subpackage ``pyscaffold.templates``,
```

Comment:
This seem to include the Python code under `pyscaffold/templates/__init__.py` itself. Is that the intention?

I think the best would be making only the `.template` files 0BSD. (Not that it differs to much from MIT, but it is nice to have the PyScaffold authorship mentioned in the case someone copies the `__init__.py` code.

Reply:
```suggestion
    The source code of PyScaffold itself is MIT-licensed with the exception of the `*.template` files under the ``pyscaffold.templates`` subpackage,
```